manim/camera/moving_camera.py

```diff
@@ -9,9 +9,14 @@
 __all__ = ["CameraFrame", "MovingCamera"]
 
 
+import math
+import operator
+
+from numpy import string_
+
 from .. import config
 from ..camera.camera import Camera
-from ..constants import ORIGIN
+from ..constants import *
```

Comment:
I don't like *-imports in library files.
```suggestion
from ..constants import LEFT, RIGHT, UP, DOWN, ORIGIN
```